### PR TITLE
Nemo/Adding errors

### PR DESCRIPTION
**Pull request by nemodjuric**
Connects to #1716 
caseflow certification 1

Note: When verifying, make sure in none of the error page cases, there is no either Header or ProgressBar component shown (refer to pic above). On the other side, during the normal workflow, Header and ProgressBar should be present at all times! 
Verify normal workflow until reaching 'Success' page. 
To cause error page - see developer.

### CODE REVIEW

File: client/app/certification/ErrorNotice.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+const UnconnectedErrorNotice = ({
+  error
+}) => {
+
+  let message;
+
+  const vbmsMessage = <main className="usa-grid">
```
Comment (amprokop) [inline client/app/certification/ErrorNotice.jsx]: it'd be more conventional and readable if you did something like

```
if ( error.includes('vbms_error') {
  return <main className="usa-grid">
   ....
  </main>;
}

return <main className="usa-grid">
  ...
</main>
```

Comment (amprokop) [inline client/app/certification/ErrorNotice.jsx]: could eliminate 10 lines of code i think

**Comment by amprokop** [comment]
Hmm! There's a lot of complexity in this PR that we could maybe avoid if we used a redirect instead of directly rendering this component in each subcomponent.

I'd suggest a new approach: 
- The `ErrorNotice` component should be rendered by a route in  `Certification.jsx`, perhaps just `/error`.
- In the individual components (like this one), if updates fail, we should `return <Redirect to="/error">` instead of rendering the ErrorNotice component.
- In `CertificationProgressBar` and `Header`, we can just check if we're on the `/error` route and hide if so, instead of passing in Redux props
- We can then remove the `toggleHeader` function everywhere. The only time we might want to hide the header and the progress bar is on the error page—since it's kind of a one off case, so it doesn't seem necessary to add all this Redux code
- We can also remove the `showHeader` prop everywhere
- Instead of creating a new action called `UPDATE_ERROR_NOTICE`, why don't we use the `CERTIFICATION_UPDATE_FAILURE` action —  that's what it's there for.

File: app/controllers/certifications_controller.rb [hunk 2]
```diff
@@ -20,7 +20,7 @@ def new
     end
   end
 
-  def update_v2_params
+  def update_certification_from_v2_form
```
Comment (nemodjuric) [inline app/controllers/certifications_controller.rb]: Yes, clarity purposes.

**Comment by nemodjuric** [comment]
It would simplify things a lot and I already thought about this. The reason I decided not to was that refresh page (since we're mimicking v1 page) won't work.

**Comment by amprokop** [comment]
@nemodjuric — can you explain what you mean by "refreshing the page won't work"?

**Comment by nemodjuric** [comment]
If you look at the VBMS failure error page, you will see a link to refresh page.

**Pull request merged**